test/e2e/specs/editor/various/adding-inline-tokens.spec.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import path from 'path';
+import fs from 'fs';
+import os from 'os';
+import { v4 as uuid } from 'uuid';
+
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'adding inline tokens', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'should insert inline image', async ( {
+		page,
+		editor,
+		pageUtils,
+	} ) => {
+		// Create a paragraph.
+		await page.click( 'role=button[name="Add default block"i]' );
```

Comment:
```suggestion
		await editor.canvas.click( 'role=button[name="Add default block"i]' );
```

I think this will fix the test failure.